range.c

```diff
@@ -1017,6 +1017,30 @@ range_to_a(VALUE range)
     return rb_call_super(0, 0);
 }
 
+/*
+ *  call-seq:
+ *    to_a -> set
+ *
+ *  Returns a set containing the elements in +self+, if a finite collection;
+ *  raises an exception otherwise.
+ *
+ *    (1..4).to_set     # => Set[1, 2, 3, 4]
+ *    (1...4).to_set    # => Set[1, 2, 3]
+ *    ('a'..'d').to_set # => Set["a", "b", "c", "d"]
+ *    ('a'..).to_set # => raises RangeError
```

Comment:
```suggestion
 *    ('a'..).to_set    # RangeError
```

The use of `# =>` in example code implies a returned result. You also want `#` to line up.